### ISSUE TITLE
Nuxt build fails with 2.3.0 release

### ISSUE THREAD

**Issue by elrolito** [nuxt-community/nuxt-i18n#28]
Trying to update to latest release causes nuxt build to fail on missing dependencies, but version 2.2.4 does not have this issue. Also seems like the build is much slower.

**Comment by paulgv** [comment]
Hi @elrolito,
Is there an error message about those missing dependencies?
Have you tried removing your `node_modules/` and reinstalling them fresh?
Also, how did you configure the module?
Thanks!

**Comment by FransTwisk** [comment]
Same here:

```
 ERROR  Failed to compile with 128 errors                                                                              9:32:49 AM

These dependencies were not found:

* aws-sdk in ./node_modules/fsevents/node_modules/node-pre-gyp/lib/unpublish.js, ./node_modules/fsevents/node_modules/node-pre-gyp/lib/publish.js and 1 other
* child_process in ./node_modules/fsevents/node_modules/detect-libc/lib/detect-libc.js, ./node_modules/fsevents/node_modules/node-pre-gyp/lib/testbinary.js and 12 others
* fs in ./server.js, ./node_modules/@nuxtjs/youch/src/Youch/index.js and 94 others
* module in ./node_modules/nuxt/lib/core/nuxt.js, ./node_modules/vue-server-renderer/build.js and 1 other
* net in ./node_modules/express/lib/request.js, ./node_modules/forever-agent/index.js and 5 others
* readline in ./node_modules/friendly-errors-webpack-plugin/src/output.js
* tls in ./node_modules/forever-agent/index.js, ./node_modules/fsevents/node_modules/forever-agent/index.js and 2 others

To install them, you can run: npm install --save aws-sdk child_process fs module net readline tls
```
Happens even after removing `node_modules` and fresh install.

**Comment by RomainLK** [comment]
I'm also having issues building both in nuxt dev and nuxt build ( nuxt build even kills the encoding of my terminal )

```
 ERROR  Failed to compile with 9 errors                                                                                                 10:43:47

These dependencies were not found:

* fs in ./node_modules/fs.realpath/index.js, ./node_modules/fs.realpath/old.js and 6 others
* module in ./node_modules/vue-server-renderer/build.js

To install them, you can run: npm install --save fs module
```

Tried to run the npm command unconvinced that it would solve anything. The error message just changed:

```
 ERROR  Failed to compile with 16 errors                                                                                                10:51:11

This dependency was not found:

* fs in ./node_modules/clone-stats/index.js, ./node_modules/convert-source-map/index.js and 14 others

To install it, you can run: npm install --save fs
```

Both `seo` and `lazyLoading` were set to `false` in config

**Comment by RomainLK** [comment]
Locked down the culprit: the async import in `i18n.utils.js`. No build error when removed.

**Comment by paulgv** [comment]
Thanks for the help guys, should be fixed in 2.3.2.
The issue will still occur when setting `langDir` to an empty string though (or pretty much any path matching root dir), this dynamic import error is a mystery to me, I'll need to give it a closer look and come up with a better fix.

**Comment by Hexodus** [comment]
I had recently a similar error because of:
`module.exports = {   I18N }`
I changed it to: 
`module.exports = I18N `
And it works ;)
